GLMakie/src/screen.jl

```diff
@@ -58,6 +58,47 @@ function Makie.insertplots!(screen::GLScreen, scene::Scene)
     foreach(s-> insertplots!(screen, s), scene.children)
 end
 
+function Base.delete!(screen::Screen, scene::Scene)
+    if !isempty(scene.children)
+        delete!.((screen,), scene.children)
```

Comment:
I think I'd prefer a for loop here for compilation, and since broadcasting creates a result vector.